### PR TITLE
editorconfig_def

### PR DESCRIPTION
**Pull request by sWoRm**
A bit a cleanup & defining multi IDE code style config

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,11 @@
+root = true
```
Comment (dcaviedesm) [inline .editorconfig:1]: I think particular editor things shouldn´t be part of code, that´s why I excluded Eclipse files. What do you think?

File: .gitignore [hunk 2]
```diff
@@ -32,7 +32,6 @@ hs_err_pid*
 target
 
 ### Eclipse template
-.metadata
```
Comment (dcaviedesm) [inline .gitignore:35]: Why did you delete this? Did you pull master to your branch before PR?

File: .editorconfig (hunk 1 above)
Comment (sWoRm) [reply .editorconfig:1]: I'm not including any specific IDE configuration but a generic one.
Please have at https://editorconfig.org/

You will learn what it is and what it does :)

File: .gitignore (hunk 2 above)
Comment (sWoRm) [reply .gitignore:35]: it's duplicated!
Have a look at line 46:

https://github.com/dcaviedesm/jazz-porque-si/blob/f5aad54c49d5eec1e165d0da88fc34caca1037d5/.gitignore#L46
